### PR TITLE
Update GithubEventService

### PR DESCRIPTION
**Pull request by smisidjan**

### CODE REVIEW

File: src/Service/ImportResourcesService.php [hunk 1]
```diff
@@ -463,6 +463,41 @@ public function importApplication(array $application, array $configuration): ?Ob
     }//end importApplication()
 
 
+    /**
```
Comment (WilcoLouwerse) [inline src/Service/ImportResourcesService.php]: Missing function description in your docblock

File: src/Service/ImportResourcesService.php [hunk 2]
```diff
@@ -463,6 +463,41 @@ public function importApplication(array $application, array $configuration): ?Ob
...
+        // Do we have a source?
+        $source           = $this->resourceService->getSource($configuration['githubSource'], 'open-catalogi/open-catalogi-bundle');
+        $componentSchema  = $this->resourceService->getSchema($configuration['componentSchema'], 'open-catalogi/open-catalogi-bundle');
+        $componentMapping = $this->resourceService->getMapping($configuration['componentMapping'], 'open-catalogi/open-catalogi-bundle');
+
+        if ($source === null
+            || $componentSchema === null
+            || $componentMapping === null
+        ) {
+            return null;
```
Comment (WilcoLouwerse) [inline src/Service/ImportResourcesService.php]: You could log an error here, or any other way of providing user feedback

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: src/Service/GithubEventService.php [hunk 3]
```diff
@@ -218,38 +164,31 @@ public function importComponentsThroughRepo(ObjectEntity $repository, string $re
...
         $repositorySchema = $this->resourceService->getSchema($this->configuration['repositorySchema'], 'open-catalogi/open-catalogi-bundle');
         $mapping          = $this->resourceService->getMapping($this->configuration['repositoryMapping'], 'open-catalogi/open-catalogi-bundle');
 
-        // Get repository from github.
-        $repositoryArray = $this->githubApiService->getRepository($name, $source);
-        if ($repositoryArray === null) {
-            // Return error if repository is not found.
-            $this->data['response'] = new Response('Cannot find repository with url: '.$repositoryUrl, 404);
+        $this->configuration['organisationSchema']  = $this->configuration['organizationSchema'];
+        $this->configuration['organisationMapping'] = $this->configuration['organizationMapping'];
```
Comment (rjzondervan) [inline src/Service/GithubEventService.php:172]: Note: discussed this with @smisidjan, this is a temporary solution as things are in flux.

**Review by rjzondervan — APPROVED** [review]
Wilco's points are fair, but seems alright overall

**Review by rjzondervan — APPROVED** [review]

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**